frontend/src/components/Dashboard/DashboardMessages.vue

```diff
@@ -3,37 +3,37 @@
         <div class="card contain-card text-black-50 bg-white mb-4">
             <div class="card-header">Announcements</div>
             <div class="card-body pt-0">
-        <table class="table table-striped">
-            <thead>
-            <tr>
-                <th scope="col">Title</th>
-                <th scope="col" class="d-none d-md-table-cell">Service</th>
-                <th scope="col" class="d-none d-md-table-cell">Begins</th>
-                <th scope="col"></th>
-            </tr>
-            </thead>
-            <tbody>
+              <table class="table table-striped">
```

Comment:
I'm unsure if the files should be reformatted in this commit as it makes reviews harder.

Reply:
You are right, the commits are now separate.